rts/motoko-rts/src/region.rs

```diff
@@ -648,18 +674,23 @@ pub(crate) unsafe fn region_migration_from_v2plus_into_v2<M: Memory>(mem: &mut M
 //
 // region manager migration/initialization, with pre-existing stable data.
 //
-// Valid cases:
-//  - from version 1 into version 2.
-//  - from version 2 into version 2.
-//
 #[ic_mem_fn]
-pub(crate) unsafe fn region_init<M: Memory>(mem: &mut M, from_version: i32) {
+pub(crate) unsafe fn region_init<M: Memory>(mem: &mut M, use_stable_regions: i32) {
```

Comment:
Detail: Maybe the `bool` type could be declared for `use_stable_regions` (although it is an integer passed by the compiler-generated code).

Reply:
Yeah, we don't seem to use strong types across the boundary. I made it a u32 for more consistency at least.